### PR TITLE
Logging for push_to_ccos

### LINKED ISSUE #63: Improvements to logging
## Problem
Currently the workflow scripts, notably those in `push_data_to_ccos/` and `sync_community_team/` are riddled with `print()` statements serving the function of logging. While this is a perfectly workable solution, it is by no means elegant. Also the process of manually indenting the messages in the `print()` statements is messy and error-prone and makes for hard to comprehend logs.

## Alternatives
Python comes with a nifty [`logging` module](https://docs.python.org/3/howto/logging.html) that should be used instead. Sprinkle some ANSI colouring and 🤯 .

## Implementation

- [ ] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by rajdesai24**
Fixes #63 
Includes logging for push_to_ccos
Removed the print statements and replaced them with a proper format based logging system.

### CODE REVIEW

File: push_data_to_ccos/get_community_team_data.py [hunk 1]
```diff
@@ -50,15 +55,15 @@ def generate_databag():
...
-    print("Pulling from Asana and generating databag...")
+    logger.log(logging.INFO, "Pulling from Asana and generating databag...")
     databag = {"projects": [], "community_builders": []}
 
     members = ASANA_CLIENT.tasks.find_by_section(
         ASANA_PROJECT_GID, opt_fields=["name", "custom_fields"]
     )
-    print("    Team members pulled.")
+    logger.log(logging.INFO, "    Team members pulled.")
+    logger.log(logging.INFO, "    Processing team members...")
```
Comment (dhruvkb) [inline push_data_to_ccos/get_community_team_data.py:65]: The main advantage of using the customised logger is to not have to indent log messages manually. The logger will take care of indentation automatically.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Apart from the manual indentation, which is not necessary with the new logger, the rest looks fine to me.

File: push_data_to_ccos/get_community_team_data.py (hunk 1 above)
Comment (rajdesai24) [reply push_data_to_ccos/get_community_team_data.py:65]: Yes, I will do that immediately. Also, the logger is really good it almost covers everything.

**Comment by rajdesai24** [comment]
@dhruvkb please check the latest commit it contains everything in the form of standard logging, do let me know if any issues. Thanks.

File: normalize_repos/normalize_repos.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
 import sys
 import traceback
 
-# Third-party
```
Comment (dhruvkb) [inline normalize_repos/normalize_repos.py:15]: This was fine, no need to change this.

File: push_data_to_ccos/get_community_team_data.py [hunk 3]
```diff
@@ -50,15 +55,15 @@ def generate_databag():
...
     databag = {"projects": [], "community_builders": []}
 
     members = ASANA_CLIENT.tasks.find_by_section(
         ASANA_PROJECT_GID, opt_fields=["name", "custom_fields"]
     )
-    print("    Team members pulled.")
+    logger.log(logging.INFO, "Team members pulled.")
+    logger.log(logging.INFO, "Processing team members...")
 
-    print("    Processing team members...")
```
Comment (dhruvkb) [inline push_data_to_ccos/get_community_team_data.py:61]: Please align logging to the correct block.

File: push_data_to_ccos/get_community_team_data.py [hunk 4]
```diff
@@ -92,9 +97,8 @@ def generate_databag():
                         }
                     )
                     break
-
-    print("    Done.")
-    print("Pull successful.")
+    logger.log(logging.INFO, "Done.")
+    logger.log(log.SUCCESS, "Pull successful.")
```
Comment (dhruvkb) [inline push_data_to_ccos/get_community_team_data.py:101]: Combine into a single `SUCCESS` log.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Looks fine for the most part, except some changes. I've marked the changes in a single location but please find other places in the PR where the same changes apply and make them there as well.

**Comment by rajdesai24** [comment]
@dhruvkb  was the last commit reviewed?
I couldn't find a response on slack

**Comment by dhruvkb** [comment]
@rajdesai24 no I have not reviewed this PR after the last review. Will do it as soon as I have some time. Feel free to work on other issues in the meanwhile.

**Comment by rajdesai24** [comment]
Sure @dhruvkb I will going for the other issues, thanks for the support.

**Comment by rajdesai24** [comment]
Sure @dhruvkb I'll be going for other issues, thanks for the support.

**Pull request closed without merging**